src/components/Calculator.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
A `Calculator.js` file inside the components directory 🚀

Reply:
@mateo951 I used JSX because this is going to return a JSX element, so, could you please tell me if should I use .js or .jsx extension?